runtime/compiler/control/J9Options.cpp

```diff
@@ -903,6 +905,8 @@ TR::OptionTable OMR::Options::_feOptions[] = {
         TR::Options::setStaticNumeric, (intptr_t)&TR::Options::_numDLTBufferMatchesToEagerlyIssueCompReq, 0, "F%d", NOT_IN_SUBSET},
    {"numInterpCompReqToExitIdleMode=", "M<nnn>\tNumber of first time comp. req. that takes the JIT out of idle mode",
         TR::Options::setStaticNumeric, (intptr_t)&TR::Options::_numFirstTimeCompilationsToExitIdleMode, 0, "F%d", NOT_IN_SUBSET },
+   {"oldAge=", " \tDefines what an old entry means when memory is low", 
```

Comment:
`oldAge` is not used hen memory is low. We need to options here one that corresponds to `OLD_AGE` and another one that corresponds to `OLD_AGE_UNDER_LOW_MEMORY`